build.xml

```diff
@@ -412,6 +412,7 @@
           <dependency groupId="org.xerial.snappy" artifactId="snappy-java" version="1.1.2.6"/>
           <dependency groupId="org.lz4" artifactId="lz4-java" version="1.4.0"/>
           <dependency groupId="com.ning" artifactId="compress-lzf" version="0.8.4"/>
+          <dependency groupId="com.github.luben" artifactId="zstd-jni" version="1.3.5-2"/>
```

Comment:
Nit: might be worth grabbing the latest version that was recently released. I think latest is 1.3.7-2 or 3.